styles/main.css

```diff
@@ -91,139 +91,76 @@ a {
     top: 5px;
 }
 
-#nav a {
-    color: #fff;
+a {
+    color: #897F7B;
 }
 
-#nav a:hover {
-    color: #D6E8D9;
+a:hover {
+    color: #E94E1B;
 }
     
 #about-page .active,
-#illustration-page .active,
+#work-page .active,
 #webdesign-page .active {
-    color: #D6E8D9;
+    color: #E94E1B;
 }
 
 
-
-.banner img {
-    width: 100%;
-    max-height: 15px;
-}
-
-#content {
-    width: 980px;
+.content {
     margin: auto;
     overflow: hidden;
     clear: both;
 }
 
 #footer {
-    max-height: 15px;
     width: 100%;
     position: absolute;
     left: 0;
     bottom: 0;
 }
 
-/* SPLASH --------------------------------*/
-
-.radius {
-    border-radius: 50%;
-    width: 250px;
-    height: 250px;
-    background-color: #f2f2f2;
-    border: 1px solid #ccc;
-}
-
-
-#splash-page {
-    background: url("../images/splashbg.png") repeat fixed;
-}
-
-#splash-page #content2 {
-    margin: 140px auto 0;
-    width: 900px;
-    text-align: center;
-}
-
-#splash-logo {
-    display: block;
-    width: 300px;
-    height: 100px;
-    margin: auto;
-}
-
-#splash-logo .splash-text {
-    padding-top: 10px;
-    line-height: 25px;
-    font-size: 20px;
-    width: 300px;
-}
-
-
-#splash-buttons {
-    left: 5px;
-    display: inline-block;
-    margin: 65px;  
-}
-
-#splash-buttons:hover {
-    opacity: 0.7;
-}
-
-
-.splash-text {
-    font-variant: small-caps;
-    font-size: 20px;
+.copyright p {
```

Comment:
see my comments in your html pages. you don't need both a footer and a copyright class just to stylize one element